documentation/alert.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../library/lib.css" />
+    <link rel="stylesheet" href="./doc.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Alert</title>
+  </head>
+  <body>
+    <div class="alert-container">
+      <h1 class="lg-title">Alert</h1>
+      <p class="sm-text">
+        An alert displays a short, important message in a way that attracts the
+        user's attention without interrupting the user's task.
+      </p>
+      <p class="md-text">Filled Alert</p>
+      <p></p>
+      <div class="filled-alert">
+        <div class="alert alert-danger-filled">
+          <i class="fas fa-exclamation-triangle alert-icon"></i>
+          <p class="sm-text bold-text">
+            Alert on eating Fires without Soft Drink
```

Comment:
Humza here it should be Fries I guess in place of fires😅

Reply:
Oh yes. Will push it on the next PR. Thanks @pradipgharge